geode-core/src/distributedTest/java/org/apache/geode/internal/cache/control/RebalanceOperationComplexPart2DistributedTest.java

```diff
@@ -220,21 +209,13 @@ protected void doRebalance(ResourceManager manager)
   /**
    * Get the bucket count for the region in the redundancy zone
    *
-   * @param regionName - name of the region to get the bucket count of
-   * @param zoneName - redundancy zone for which to get the bucket count
    * @return - the total bucket count for the region in the redundancy zone
    */
-  protected int getZoneBucketCount(String regionName, String zoneName) {
+  protected int getZoneBucketCount() {
```

Comment:
This method name should probably be changed to "getZoneABucketCount()"

Reply:
Fair. Done.